tests/subsys/logging/log_switch_format/overlay_deferred.conf

```diff
@@ -1,4 +1,5 @@
 CONFIG_LOG_MODE_IMMEDIATE=n
+CONFIG_ZTEST_NEW_API=y
```

Comment:
I don't think you need to add this option to overlays for deferred and immediate mode as you have already added it to prj.conf.

Reply:
> I don't think you need to add this option to overlays for deferred and immediate mode as you have already added it to prj.conf.

ok,I will remove it